.github/workflows/windows-docker-image.yaml

```diff
@@ -92,12 +93,20 @@ jobs:
           docker pull ${gh_user}/rez-win-base:${ENV:LAST_DOCKER_BASE_REVISION}
           docker tag ${gh_user}/rez-win-base:${ENV:LAST_DOCKER_BASE_REVISION} rez-win-base:${ENV:LAST_DOCKER_BASE_REVISION}
 
-      - name: Build and push Docker image
+      - name: Build Docker image if needed
         run: |
-          ${gh_user} = ("${{ github.repository }}" -Split '/')[0]
-          ${ENV:DOCKER_TAG} = "${{ github.sha }}".Substring(0, 8)
           ${Env:LAST_DOCKER_BASE_REVISION} = $( git log -n 1 --author-date-order --pretty=format:%H -- .\.github\docker\rez-win-base\  .\.github\workflows\windows-docker-image.yaml ).SubString(0, 8)
-          cd .github\docker\rez-win-py
-          docker build --tag ${gh_user}/rez-win-py:${{ matrix.python-version }}-${ENV:DOCKER_TAG} --build-arg PYTHON_VERSION="${{ matrix.python-version }}" --build-arg BASE_TAG=${ENV:LAST_DOCKER_BASE_REVISION} .
-          docker push ${gh_user}/rez-win-py:${{ matrix.python-version }}-${ENV:DOCKER_TAG}
-
+          ${Env:LAST_DOCKER_PY_REVISION} = $( git log -n 1 --author-date-order --pretty=format:%H -- .\.github\docker\rez-win-py\  .\.github\workflows\windows-docker-image.yaml ).SubString(0, 8)
```

Comment:
Doesn't this need to also include .\.github\docker\rez-win-base\, so that an update to the base image causes the python images to be rebuilt also?

Reply:
I think you are right.